packages/actor-rdf-parse-html-script/lib/ActorRdfParseHtmlScript.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ActorRdfParseFixedMediaTypes,
+  IActionRdfParse,
+  IActionRootRdfParse,
+  IActorOutputRootRdfParse,
+  IActorRdfParseFixedMediaTypesArgs,
+  IActorRdfParseOutput,
+  IActorTestRootRdfParse,
+} from "@comunica/bus-rdf-parse";
+import {ActionContext, Actor, Mediator} from "@comunica/core";
+import {Readable} from "stream";
+
+/**
+ * A HTML RDF Parse actor that listens on the 'rdf-parse' bus.
+ *
+ * It is able to extract JSON-LD from HTML files and parse the JSON-LD based RDF serializations
+ * and announce the presence of them by media type.
+ */
+export class ActorRdfParseHtmlScript extends ActorRdfParseFixedMediaTypes {
+
+  public mediatorRdfParse: Mediator<Actor<IActionRootRdfParse, IActorTestRootRdfParse,
+    IActorOutputRootRdfParse>, IActionRootRdfParse, IActorTestRootRdfParse, IActorOutputRootRdfParse>;
+
+  constructor(args: IActorRdfParseFixedMediaTypesArgs) {
+    super(args);
+  }
+
+  public async runHandle(action: IActionRdfParse, mediaType: string, context: ActionContext):
+    Promise<IActorRdfParseOutput> {
+
+    const myQuads = new Readable({objectMode: true});
```

Comment:
Unclear variable name, I propose to change this to just `quads`. (then you can also abbreviate the object on line 90)

Reply:
Also, make sure to write your objects like this `{ value: true }` instead of this `{value:true}`. This applies to multiple places in this PR.